src/material/core/datetime/date-selection.ts

```diff
@@ -0,0 +1,159 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {DateAdapter} from '@angular/material/core';
+import {Subject} from 'rxjs';
+
+export abstract class MatDateSelection<D> {
+  valueChanges = new Subject<void>();
```

Comment:
it might make sense to do:
```ts
private _valueChangesSubject = new Subject<void>();

valueChanges = this._valueChangesSubject.asObservable();
```

That way people can't access the subject and emit things on it